supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+[[partner-and-hybrid-cloud-infrastructure-conventions]]
+
+[discrete]
+[[access-mode]]
+==== image:images/yes.png[yes] access mode (noun)
+*Description*: The concurrency method used when accessing a storage component, such as `readwriteonce` or `readwritemany`.
+
+*Use it*: Yes
+
+
+[discrete]
+[[availability]]
+==== image:images/caution.png[with caution] availability (adjective)
+The state of being available to perform work in a system, in the context of high availability or availability zones. The ability of a component to be utilized at a given time.
```

Comment:
This is missing the `*Description*:` label

Reply:
Fixed!